setup.py

```diff
@@ -26,11 +26,8 @@
     python_requires=">=3.7",
     install_requires=[
         "flatten-dict==0.4.2",
-        "jupyter==1.0.0",
-        "matplotlib>=3.5.3",
         "pandas>=0.25.2",
         "requests==2.31.0",
-        "statsmodels==0.13.2",
         "tqdm==4.64.1",
         "pydantic==1.10.2",
```

Comment:
Not sure where we are using pydantic but binding it to a specific version is a bad idea.